obstack/obstack.h

```diff
@@ -78,7 +78,7 @@
    When the chars burst over a chunk boundary, we allocate a larger
    chunk, and then copy the partly formed object from the end of the old
    chunk to the beginning of the new larger chunk.  We then carry on
-   accreting characters to the end of the object as we normally would.
+   accrediting characters to the end of the object as we normally would.
```

Comment:
I think this is wrong change. I think this word exists - it's derived from accretion.

Reply:
@t8m my bad, thanks for catching this (and the detailed review in general) :+1: Fixed.